### PR TITLE
add getDepositHistory

### PR DESCRIPTION
**Pull request by akaster99**
Add a method (getDepositHistory) and types

### CODE REVIEW

File: src/credit.ts [hunk 1]
```diff
@@ -267,4 +268,21 @@ export default class Credit extends AinftBase {
     const trailingUrl = `deposit/transaction`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+
+    /**
```
Comment (platfowner) [inline src/credit.ts:272]: indentation?

File: src/credit.ts [hunk 2]
```diff
@@ -267,4 +268,21 @@ export default class Credit extends AinftBase {
...
+   * send deposit transaction to nft server
```
Comment (platfowner) [inline src/credit.ts:273]: send -> Send?
nft server -> AINFT Server?

period(.) at the end?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/credit.ts (hunk 1 above)
Comment (akaster99) [reply src/credit.ts]: I applied it. thanks!

File: src/credit.ts (hunk 2 above)
Comment (akaster99) [reply src/credit.ts]: I applied it. thanks!

**Pull request merged**